stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,209 @@
+  return (
+    <Header
+      appLogo={
+        <HeaderLogo
+          logo={
+            <svg
+              viewBox='0 0 16 16'
+              xmlns='http://www.w3.org/2000/svg'
+              aria-hidden='true'
+            >
+              <path d='m12.6 13.4c-1.2-1.5-2.1-3.1-2.4-5.5-2.7 3.9-4.6 4.2-5.7 2.4l-1.2 5.7h-2.2l3.5-14.1 1.8-.4c-.1.5-1.4 6.2.6 7 2 .7 4.6-8.5 4.6-8.5l2.2.4c-1.6 3.7-1.6 7.6 1.1 10.9l-2.3 2.1' />
+            </svg>
+          }
+          onClick={() => action('Clicked on the title')()}
+        >
+          Microstation
+        </HeaderLogo>
+      }
+      breadcrumbs={
+        <HeaderBreadcrumbs
+          items={[
+            <HeaderButton
+              key='project'
+              menuItems={buildMenu('Project')}
+              name={'Project A (Super Size Edition)'}
+              description={'YJC-2249'}
+              startIcon={<SvgNetwork />}
+            />,
+            <HeaderButton
+              key='iModel'
+              menuItems={buildMenu('IModel')}
+              name={'IModel B'}
+              startIcon={
+                <img src='https://itwinplatformcdn.azureedge.net/iTwinUI/stadium.png' />
+              }
+              isActive={true}
+            />,
+            <HeaderButton
+              key='version'
+              name={'Version C'}
+              onClick={() => action('Clicked on the Version')()}
+              startIcon={<SvgVersion />}
+            />,
+          ]}
+        />
+      }
+      actions={[
+        <IconButton
+          key='notif'
+          onClick={() => action('Clicked on the notification bell')()}
+          styleType='borderless'
+        >
+          <SvgNotification />
+        </IconButton>,
+        <DropdownMenu key='help' menuItems={buildMenu('Help')}>
+          <IconButton styleType='borderless'>
+            <SvgHelpCircularHollow />
+          </IconButton>
+        </DropdownMenu>,
+      ]}
+      userIcon={
+        <DropdownMenu menuItems={buildMenu('User')}>
+          <IconButton styleType='borderless'>
+            <UserIcon
+              size='medium'
+              abbreviation='TR'
+              backgroundColor={getUserColor('Terry Rivers')}
+              image={
+                <img src='https://itwinplatformcdn.azureedge.net/iTwinUI/user-placeholder.png' />
+              }
+              title='Terry Rivers'
+            />
+          </IconButton>
+        </DropdownMenu>
+      }
+      menuItems={buildMenu('More')}
+      {...args}
+    >
+      <div>
+        <style>{`.iui-slim .center-input { padding-top: 0; padding-bottom: 0}`}</style>
+        <Input
+          className='center-input'
+          placeholder='Search within Model Beta...'
+          style={{
+            borderRadius: 22,
+            width: '20vw',
+            margin: 12,
+            transition: 'all 0.2s ease',
+          }}
+        />
+      </div>
+    </Header>
+  );
+};
+
+export const basic: Story<HeaderProps> = (args) => {
```

Comment:
This one and another one story looks odd to me. Maybe we should have a story without breadcrumbs.

Reply:
Yeah..Do not look really useful.. I guess just have a story with simple buttons (not dropdowns) in breadcrumbs and you can have on the right side only user-icon lets say..
SOmething from more used examples and that should be enough.

Reply:
Well, I dont really see the use of those 2 different stories either, I dont have example of "more used", the full one is the recommended approach. The Full example already shows a button with no dropdown with the "Version" menu. I can addit to the "SimpleUserIcon" and name this other one the "Basic" ?

Reply:
I dont have a story with the translatedStrings, but, since this is only changing the aria-label, I'm not sure its worth a story.